app/helpers/moments_helper.rb

```diff
@@ -15,8 +15,25 @@ def edit_moment_props
     edit_form_props(moment_form_inputs, moment_path(@moment))
   end
 
+  def secret_share_url_props(moment)
```

Comment:
why is this here just to call a private method? why not expose that method?

Reply:
To protect this trivial logic and future logic being used to generate the props as well as to keep the style of the file in line with the other functions in this helper. Is there a reason not to? style or security?

Reply:
I'd say more in terms of OO design. Generally private methods are in place to hide implementation details that aren't necessary for the public interface of an object, but since the only thing this method does is call a private method (which returns a hash, so the internals of the method are still exposed) it doesn't necessarily need to be private. Also, in ruby, `private` methods shouldn't really be used as a form of security since they aren't really private as in other OO languages